Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -141,6 +141,12 @@ def audit
       args.named.to_formulae_and_casks
           .partition { |formula_or_cask| formula_or_cask.is_a?(Formula) }
     end
+
+    if audit_formulae.empty? && audit_casks.empty?
+      ofail "No audits to perform"
```

Comment:
```suggestion
      ofail "No formulae or casks specified to audit!"
```

Reply:
I am fine with changing the wording, but I think this might cover the situation better:
```suggestion
      ofail "No matching formulae or casks to audit!"
```

Reply:
@SMillerDev Works for me!